charging_area:
  upper_left: [-1,1]
  down_right: [1,-1]